test/Tryouts/Corax/IndexReader.cs

```diff
@@ -47,6 +47,22 @@ private unsafe long GetStringId(TransactionOperationContext context, Table strin
             }
         }
 
+        
+        public unsafe int GetTermFreq(TransactionOperationContext context, string term)
+        {
+            var stringsTable = context.Transaction.InnerTransaction.OpenTable(IndexBuilder.StringsTableSchema, "Strings");
```

Comment:
Probably better to add `stringsTable` as a field to the reader, instead of calling `OpenTable` all the time